### ISSUE TITLE
Discord Chat Group

### ISSUE THREAD

**Issue by ZeyadAhmed** [ComponentFactory/Krypton#188]
http://discordapp.com is very known chatting platform used by programmers and gamers. many projects use it for helping and discussing, this project is amazing and I think it deserves one.

**Comment by Wagnerp** [comment]
Hi @ZeyadAhmed 

That's a great idea I'm wondering if I should create one for my & @Smurf-IV forks?

**Comment by ZeyadAhmed** [comment]
Sorry for late reply.
Yeah it will be cool, I have many questions about Krypton and I think having a discord server for instant messaging will amazing, I think we should need for someone to make a one and invite us.

**Comment by Wagnerp** [comment]
Hi @ZeyadAhmed 

You can join our server by following the link I've put [here](https://github.com/Wagnerp/Krypton-NET-5.470)

**Comment by ZeyadAhmed** [comment]
Cool, I'm there. But it's not that active though.

**Comment by Wagnerp** [comment]
No, but it's quite new may take time for others to notice, join and get used to.
